src/fios.h

```diff
@@ -16,6 +16,7 @@
 #include "gamelog.h"
 #include "network/core/tcp_content_type.h"
 #include "timer/timer_game_calendar.h"
+#include <map>
```

Comment:
I think we should add this to `stdafx.h`. As the fact it works for some platforms and not for others is just annoying, and we shouldn't want to deal with :)